index.html

```diff
@@ -163,6 +163,38 @@ <h3>Production packages</h3>
     <h2>Who we work with</h2>
     <a name="Customers"></a>
   </div>
+  <div class="bg-accent-warm-light padding-4">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-2"></div>
+      <div class="tablet:grid-col-2">
+        <img src="{{ site.baseurl }}/assets/images/logos/EPA-logo.svg" alt="EPA logo" class="square-10"/>
+      </div>
+      <div class="tablet:grid-col-8">
+        <h3>Environmental Protection Agency - AirNow.gov</h3>
+        <h4>EPA AirNow Application</h4>
+      </div>
+      <div class="tablet:grid-col-2"></div>
+    </div>
+    <div class="grid-row usa-prose">
+      <div class="tablet:grid-col-2"></div>
+        <div class="tablet:grid-col border-top border-base-light">
+        <p>
+          The EPA AirNow team attributed the stability of <a href="https://www.airnow.gov">airnow.gov</a> to their ability to easily scale their applications on Cloud.gov. 
```

Comment:
This sentence needs to give more context for the quote that follows for users who don't click through to the full customer story.  In one sentence, we should highlight:

- what happened ( a surge in traffic)
- how much did traffic surge
- what went well (remarkable stability for AirNow.gov)